AmpleOrganics/Blaze.vue
# fix initialization of select component

# Link to Github Issue

Did not have one

# Description

This PR makes select initialize properly without emitting an update on initialization

before: when loading ao-select the input event got triggered regardless of the scenario

now: uses the created hook instead of the mounted hook and therefore does not fire the update method in the component

#QA 
add a console.log statement in the updateInput, 
serve the docs go to select, 
a console statement should NOT exist on startup, 
switch the select box and select another value, you should then see a log

go to the select.vue file in the docs, 
set the v-modeled variable to Charmander, 
refresh and make sure that charmander is pre selected and a console.log statement does not appear in your console
